core/app/models/workarea/order/item.rb

```diff
@@ -147,13 +147,32 @@ def customized?
       customizations.present?
     end
 
+    # Determine whether the additional fields on this item are
+    # equivalent to the passed-in Hash of parameters. Uses the
+    # `distinct_order_item_attributes` configuration setting to filter
+    # down which parameters it will be checking equivalence on.
+    #
+    # @param [Hash] params - Updated order item parameters.
+    # @return [Boolean] whether this item's attributes is equal to the
+    #                   passed-in parameters.
+    def attributes_eql?(params = {})
+      equivs = Workarea.config.distinct_order_item_attributes.map(&:to_sym)
+      attrs = params&.symbolize_keys&.slice(*equivs)
+      customizations = params[:customizations]&.stringify_keys! || {}
```

Comment:
`customizations_eql?` handles the blank case, so this doesn't need a default value.